control_plane/roles/webui_awx/files/awx_operator.yml

```diff
@@ -0,0 +1,531 @@
+# This file is generated by Ansible. Changes will be lost.
```

Comment:
Is the `awx_operator.yml` code sourced from somewhere? If so, can we just deploy the operator from that source?

Reply:
@lwilson Yes but the resources will be deployed in **default** namespace. In order to group them into cusom namespace **awx** i have took the code from source and changed the namespace